bot/exts/evergreen/tic_tac_toe.py

```diff
@@ -318,9 +318,14 @@ async def show_tic_tac_toe_board(self, ctx: Context, game_id: int) -> None:
             return
         game = self.games[game_id - 1]
 
+        if game.draw:
+            description = f"{game.winner} vs {game.loser} (draw)\n\n{game.format_board()}"
```

Comment:
Just like the `.ttt history` command, there is no winner or loser in a drawn game, so the statement should use the players in the game, see: https://github.com/python-discord/sir-lancebot/blob/0a6a355419d9382b35ac651117287d907e98af0c/bot/exts/evergreen/tic_tac_toe.py#L301

Reply:
Yeah, my bad. I'll commit the changes right away. Thanks for pointing it out.